test/jdk/jdk/incubator/vector/Byte128VectorLoadStoreTests.java

```diff
@@ -23,6 +23,7 @@
 
 /*
  * @test
+ * @enablePreview
```

Comment:
Can we do the same trick you did for the stream API test, and add a new TEST.properties file in this folder and enable preview there? Or, I assume you tried and that didn't work?

Reply:
Let's refrain from adding a properties file, given that there’s a template file already that defines test configuration (where I added `@enablePreview`), test/jdk/jdk/incubator/vector/templates/X-LoadStoreTest.java.template.